component-library/components/generic/icon/icon.scss

```diff
@@ -0,0 +1,24 @@
+.c-icon {
+    &__round {
+        border-radius: 50%;
+        border-color: var(--neutral-000, #FFF);
```

Comment:
--neutral-000 needs to be changed to --brand-grey-000

Reply:
Thanks.